src/components/Confirm/index.js

```diff
@@ -84,15 +84,16 @@ const Previews = ({capture, retakeAction, confirmAction} ) =>
 
 
 export default class Confirm extends Component {
-  completeCapture(method, side, capture, confirmCapture, nextStep) {
+  completeCapture(id, method, side, capture, confirmCapture, nextStep) {
+    capture.id = id
```

Comment:
Yes, cause we need to have a capture with an id in the store and in the confirm capture we just need to pass the id